src/map/atcommand.c

```diff
@@ -1714,6 +1714,15 @@ ACMD(bodystyle)
 
 	memset(atcmd_output, '\0', sizeof(atcmd_output));
 
+	if (!((sd->job&MAPID_THIRDMASK) == MAPID_GUILLOTINE_CROSS || (sd->job&MAPID_THIRDMASK) == MAPID_GENETIC
```

Comment:
It's not exactly the same. Check the types (I'll pretend that boolean is a type)

The line before this PR was:

```
// original
! (int == int || int == int || int == int)
// simplified:
! (bool || bool || bool)
// further simplified:
! bool
```

which is pretty good, except for the ugly `(! (long expression here))` that has the problem of being counter-intuitive and makes it very easy not to see the negation.

You want to change it to:

```
// suggested line:
(int || int || int) == 0
// simplified:
bool == 0
```

which contains several types "formal" mismatch. You're doing a logical OR between integers instead of booleans. You're doing a comparison between a boolean and zero.

What I suggest is, instead:

```
// suggested line:
int != int && int != int && int != int
// simplified:
bool && bool && bool
// simplified again:
bool
```

The types there are all correct.

My reasoning assumes the following (which isn't technically true in C, since it doesn't really support the boolean type, but please pretend with me that it does, at least at a formal level)

- operator `==` and `!=` take two `int` and return a `bool`
- operator `==` and `!=` take two `bool` and return a `bool`
- using operator `==` or `!=` between a `bool` and an `int` is an error
- operator `||` and `&&` take two `bool` and return a `bool`
- using operator `||` or `&&` between one or more `int` is an error
- operator `&` and `|` take two `int`, return an `int`
- operator `!` takes a `bool` and returns a `bool`